dashboard/django/stats/views.py

```diff
@@ -120,6 +132,15 @@ def namespace(request, namespace_name):
     })
 
 
+def deleteNamespace(request, namespace_name):
+    url = settings.SERVICE_URL + '/admin/v2/namespaces/' + namespace_name
```

Comment:
This option settings.SERVICE_URL needs to be included in the configuration file settings.

Reply:
Done